server/blob/blob-api/src/test/java/org/apache/james/blob/api/MetricableBlobStoreContract.java

```diff
@@ -79,32 +57,43 @@ private void setupExpectations() {
 
     @Test
     default void saveBytesShouldPublishSaveBytesTimerMetrics() {
+        long sleepDurationInMs = Duration.ofSeconds(1).toMillis();
+
         testee().save(BYTES_CONTENT).block();
-        testee().save(BYTES_CONTENT).block();
-        verify(metricsTestExtension.saveBytesTimeMetric, times(2)).stopAndPublish();
+
+        assertThat(metricsTestExtension.getMetricFactory().executionTimesFor(SAVE_BYTES_TIMER_NAME))
+            .allSatisfy(executionInMs -> assertThat(executionInMs).isLessThan(sleepDurationInMs));
     }
 
     @Test
     default void saveInputStreamShouldPublishSaveInputStreamTimerMetrics() {
+        long sleepDurationInMs = Duration.ofSeconds(1).toMillis();
+
         testee().save(new ByteArrayInputStream(BYTES_CONTENT), BYTES_CONTENT.length).block();
-        testee().save(new ByteArrayInputStream(BYTES_CONTENT), BYTES_CONTENT.length).block();
-        testee().save(new ByteArrayInputStream(BYTES_CONTENT), BYTES_CONTENT.length).block();
-        verify(metricsTestExtension.saveInputStreamTimeMetric, times(3)).stopAndPublish();
+
```

Comment:
why did you remove those testee calls? I find it's helpful to know that metric recording is working well with multiple calls